### ISSUE TITLE
Bronken example link on docs/pipelines/resources/#pull-request-resource

### ISSUE THREAD

**Issue by flaviodsr** [tektoncd/website#158]
# Expected Behavior
The [example](https://tekton.dev/docs/pipelines/cmd/pullrequest-init/example) link on https://tekton.dev/docs/pipelines/resources/#pull-request-resource should return a valid webpage.

# Actual Behavior
The [example](https://tekton.dev/docs/pipelines/cmd/pullrequest-init/example) link on https://tekton.dev/docs/pipelines/resources/#pull-request-resource returns:

```Not found.
Oops! This page doesn't exist. Try going back to our home page.
```
# Steps to Reproduce the Problem

1. Navigate to https://tekton.dev/docs/pipelines/resources/#pull-request-resource
2. Scroll down until "For an example of the output this resource provides, see[example](https://tekton.dev/docs/pipelines/cmd/pullrequest-init/example)."
3. Click on the [example](https://tekton.dev/docs/pipelines/cmd/pullrequest-init/example) hyperlink

**Comment by jjasghar** [comment]
It seems that the file is completely missing. https://github.com/tektoncd/website/tree/master/content/en/docs/Pipelines I'm not sure where it went, nor how to remove it, being that the `_index.md` doesn't have it either.

@AlanGreene @sergetron any thoughts here?

**Comment by AlanGreene** [comment]
The pipelines docs are synced from tektoncd/pipeline, so the source is actually at https://github.com/tektoncd/pipeline/blob/master/docs/resources.md

Relative links in the docs are a known problem. In this particular case the link should probably be the fully qualified URL of the content on GitHub since there are many files involved and it's easier to review on GitHub rather than individually downloading files, opening in an editor, etc.

https://github.com/tektoncd/pipeline/blame/master/docs/resources.md#L519

The URL could be updated to point to https://github.com/tektoncd/pipeline/tree/master/cmd/pullrequest-init/example

https://github.com/tektoncd/website/issues/140 will help with some of these issues as individual projects will have more direct control over what / how content gets surfaced on the website.

**Comment by popcor255** [comment]
fixed #181 

/close
